SensusUI/PromptForInputsPage.cs

```diff
@@ -241,7 +250,47 @@ public PromptForInputsPage(InputGroup inputGroup, int stepNumber, int totalSteps
             };
 
             #endregion
-                
+
+            navigationStack.Children.Add(previousNextStack);
+
+            #region cancel button
+
+            bool cancelButtonTapped = false;
+
+            if (showCancelButton)
+            {
+                Button cancelButton = new Button
+                    {
+                        HorizontalOptions = LayoutOptions.CenterAndExpand,
+                        FontSize = 20,
+                        Text = "Cancel"
+                    };
```

Comment:
Run code reformatting on entire document to fix spacing above and below. Always good to reformat before committing.